runelite-client/src/main/java/net/runelite/client/plugins/timersandbuffs/TimersAndBuffsPlugin.java

```diff
@@ -615,6 +615,31 @@ else if (nextSuperAntifireTick - tickCount <= 0)
 			updateVarTimer(MOONLIGHT_POTION, moonlightValue, IntUnaryOperator.identity());
 		}
 
+		if (event.getVarbitId() == Varbits.STONE_OF_JAS_EMPOWERMENT && config.showOverload())
```

Comment:
Instead of hijacking overload/prayer enhance timers, let's make one new config to cover Tormented demon buffs for Stone of Jas/Smouldering heart/Smouldering gland. (Technically the Stone of Jas empowerment is not related, but you only get the buff before fighting some tds, so it's close enough)